cibuild.sh

```diff
@@ -37,20 +47,39 @@ function python-tools {
   add_path $PYTHONUSERBASE/bin
 }
 
+function flock {
+  case $os in
+    Darwin)
+      brew tap discoteq/discoteq
```

Comment:
i don't see a point to build it from source where the os provides it.

Reply:
Why not if the change can simplify the script? macOS will be pure subset of Linux if we take this approach for genromfs and flock.

Reply:
IMO it's better to test with tools people usually use. building this version of flock from source is not something linux folks do.

Reply:
Let's try our best to reuse the test list, the speration is the last resort.

Reply:
maybe you replied on a wrong thread? anyway i unified the lists as you requested.

Reply:
@xiaoxiang781216 do you still want to avoid homebrew stuff?
the latest version is using homebrew for the following tools. i guess it's easy to install flock from the source. but not for the other two.
* u-boot-tools
* flock
* x86_64-elf-gcc